src/models/db/notification.js

```diff
@@ -1,18 +0,0 @@
-'use strict'
-
-const Model = require('five-bells-shared').Model
-const PersistentModelMixin = require('five-bells-shared').PersistentKnexModelMixin
-const knex = require('../../lib/knex').knex
-
-class Notification extends Model {
```

Comment:
Doesn't the notification still need to be saved to the DB in order for it to look it up later?